build/scripts/aws/dynamodb/bulk_delete_items.py

```diff
@@ -0,0 +1,33 @@
+"""
+Bulk delete items from a DynamoDB table.
+
+The file should contain a list of items, in JSON Lines format, to delete.
+
+Example usage:
+    python3 bulk_delete_items.py my-table my-file.jsonl
```

Comment:
Documentation: can you copy the expected `jsonl` file contents from the PR description or add a small README with this info so it's not lost in the PR?

Reply:
Added it to the script doc 👍